src/cmd/MVP-4/.env

```diff
@@ -1,19 +1,19 @@
-CREDENTIALS=".\\secret.json"
+CREDENTIALS="./secret.json"
 #EXCEL_ID="1ppqYGlGTXGb-PKOcMesyvz12pLuD1XoPAJ0N_kpRJtg"
 EXCEL_ID="1BEwASubu0el9oQA6PSwVKaNU-Q6gbJ40JR6kgqguKYE"
+#EXCEL_ID="1u0cOaWQI8t7aKfZ4aQCsexKLB3rIPHqX3kDrx0QJZMU"
```

Comment:
If u r not using it, then remove it. If u need to recover it in any moment u can retrieve the value from git